src/core/polyhedral/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+IF(DEFINED CILK_PATH)
```

Comment:
isn't this going to be _always_ defined, but sometimes empty?

Reply:
fixed